src/chrome/content/rules/Touchgen.net.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Touchgen" platform="mixedcontent">
```

Comment:
Took off the mixedcontent as you requested. The previous commit did not pass the rules test. Should be fine now.